http_server/fns/data_fns.php

```diff
@@ -319,7 +319,23 @@ function is_moderator($pdo, $check_ref = true)
 
 
 
-//
+// returns true if you are logged in as an admin, false if you are not
+function is_admin($pdo, $check_ref = true)
+{
+    $is_admin = false;
+    try {
+        check_moderator($pdo, $check_ref, 3);
+        $is_admin = true;
+    } catch (Exception $e) {
```

Comment:
Bad idea to have completely empty catch blocks. But if you really aren't doing anything with the exception, don't declare it; `catch {}` is perfectly valid.